appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.controller.js

```diff
@@ -239,20 +246,104 @@ export async function postIPComment(request, response) {
  * @param {import('@pins/express/types/express.js').Request} request
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  */
-export async function renderCheckYourAnswers(request, response) {
-	const pageContent = checkYourAnswersPage(
-		request.currentAppeal,
-		request.session?.addIpComment,
-		request.session?.fileUploadInfo,
-		request.errors
-	);
+export async function renderCheckYourAnswers(
+	{
+		errors,
+		currentAppeal: { appealReference, appealId } = {},
+		currentComment: { id: commentId } = {},
+		session: {
+			fileUploadInfo: { files: [{ name, blobStoreUrl }] } = {
+				files: [{ name: '', blobStoreUrl: '' }]
+			},
+			addIpComment: {
+				[redactionStatusFieldName]: redactionStatus,
+				day,
+				month,
+				year,
+				firstName,
+				lastName,
+				emailAddress,
+				addressProvided,
+				addressLine1,
+				addressLine2,
+				town,
+				county,
+				postCode
+			} = {
+				redactionStatus: 'unredacted',
+				day: '',
+				month: '',
+				year: '',
+				firstName: '',
+				lastName: '',
+				emailAddress: '',
+				addressProvided: '',
+				addressLine1: '',
+				addressLine2: '',
+				town: '',
+				county: '',
+				postCode: ''
+			}
```

Comment:
Not keen on this but there doesn't seem to be a way to mock the session in the tests so setting defaults was the best I could come up with to make them pass

Reply:
I agree this is pretty unreadable. If it's just for the sake of tests, my preference would be to remove this and simply not test whatever becomes untestable as a result (as you say, there's no way to mock the session as far as I know)